### PR TITLE
Fix compilation of release cmake build type for ppc64le

### PR DESCRIPTION
**Pull request by pkoutsov**
This extra tiny modification of CMakeLists.txt is necessary so Hyrise can compile successfully for ppc64le arch and ```-DCMAKE_BUILD_TYPE=RelWithDebInfo```. Otherwise compilation fails with this message

```console
unrecognized command line option '-march=native'; did you mean '-mcpu=native'
```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -84,7 +84,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${FLAGS_ALL}")
 # https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/compiler-flags-across-architectures-march-mtune-and-mcpu
 # We check specifically for arm64 (e.g., Apple's M1) or aarch64 (e.g., Raspberry Pi 4), as these are the two ARM architectures we have tested so far.
 # Match for "arm" on your own risk to include other ARM architures.
-if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm64|aarch64")
+if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm64|aarch64|ppc64le")
```
Comment (Bouncner) [inline CMakeLists.txt:87]: Could you add a comment/extend the one on ARM builds? Is it similar to ARM and arch native works for GCC but not for clang?

**Comment by mrks** [comment]
@Bouncner - should we just move to https://stackoverflow.com/questions/46724267/cmake-march-hardware ?

**Comment by Bouncner** [comment]
Looks cleaner. Sure, why not.

**Comment by mrks** [comment]
Do you want to look into that or should I?

**Comment by Bouncner** [comment]
The M1 commit has been my first contact with CMake so far. I guess you're faster with that, so I'd be happy if you could do that one.

**Comment by pkoutsov** [comment]
Since there is a pending PR (https://github.com/hyrise/hyrise/pull/2313) that addresses this exact issue, please feel free to close this PR.

**Comment by mrks** [comment]
That was the plan for when we have verified that this works for all architectures. Can you confirm that the solution from #2313 also fixes the issue for ppc64le?

**Pull request closed without merging**